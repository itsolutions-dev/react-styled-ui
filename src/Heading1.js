import styled from 'styled-components';
import { getTextColor } from './utils/';

const Heading1 = styled.h1`
  font-size: ${props => props.theme.h1.fontSize};
  font-weight: ${props => props.theme.h1.fontWeight};
  line-height: ${props => props.theme.h1.lineHeight};
  color: ${props => getTextColor(props)};
`;

export default Heading1;
