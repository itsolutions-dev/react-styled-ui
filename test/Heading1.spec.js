import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import 'styled-components-test-utils/lib/jest';
import Heading1 from '../src/Heading1';

describe('Heading1', () => {
  test('should render a Button', () => {
    const component = ReactTestRenderer.create(<Heading1 />);
    expect(component).toBeDefined();
  });

  test('should have a h1 tag', () => {
    expect(Heading1).toHaveComponent('h1');
  });
});
