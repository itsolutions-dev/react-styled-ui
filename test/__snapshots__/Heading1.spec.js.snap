// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading1 should match snapshot 1`] = `
<h1
  className="sc-bdVaJa gRTzbq"
/>
`;
